presto-jmx/src/main/java/com/facebook/presto/connector/jmx/JmxConnectorFactory.java

```diff
@@ -58,32 +54,37 @@ public ConnectorHandleResolver getHandleResolver()
     @Override
     public Connector create(String connectorId, Map<String, String> properties)
     {
-        return new Connector()
-        {
-            @Override
-            public ConnectorTransactionHandle beginTransaction(IsolationLevel isolationLevel, boolean readOnly)
-            {
-                checkConnectorSupports(READ_COMMITTED, isolationLevel);
-                return JmxTransactionHandle.INSTANCE;
-            }
+        return new JmxConnector(
+                connectorId,
+                mbeanServer,
+                nodeManager,
+                getDumpTables(properties),
+                getDumpPeriod(properties),
+                getEvictionLimit(properties));
+    }
 
-            @Override
-            public ConnectorMetadata getMetadata(ConnectorTransactionHandle transactionHandle)
-            {
-                return new JmxMetadata(connectorId, mbeanServer);
-            }
+    private List<String> getDumpTables(Map<String, String> properties)
```

Comment:
extract JmxConnectorConfig class and propagate it with with Bootstrap like other connectors do.
And make "jmx.dump-period" config param a Duration.